docs/guides/setting-up-development-project.md

```diff
@@ -395,1004 +119,197 @@ npm i zeebe-node dotenv
-    }
-    public class ZeebeService: IZeebeService
+  brokers: [
     {
-        private readonly IZeebeClient _client;
-        private readonly ILogger<ZeebeService> _logger;
-
-        public ZeebeService(IEnvReader envReader, ILogger<ZeebeService> logger)
-        {
-            _logger = logger;
-            var authServer = envReader.GetStringValue("ZEEBE_AUTHORIZATION_SERVER_URL");
-            var clientId = envReader.GetStringValue("ZEEBE_CLIENT_ID");
-            var clientSecret = envReader.GetStringValue("ZEEBE_CLIENT_SECRET");
-            var zeebeUrl = envReader.GetStringValue("ZEEBE_ADDRESS");
-            char[] port =
-            {
-                '4', '3', ':'
-            };
-            var audience = zeebeUrl?.TrimEnd(port);
-
-            _client =
-                ZeebeClient.Builder()
-                    .UseGatewayAddress(zeebeUrl)
-                    .UseTransportEncryption()
-                    .UseAccessTokenSupplier(
-                        CamundaCloudTokenProvider.Builder()
-                            .UseAuthServer(authServer)
-                            .UseClientId(clientId)
-                            .UseClientSecret(clientSecret)
-                            .UseAudience(audience)
-                            .Build())
-                    .Build();
-        }
-
-        public Task<ITopology> Status()
-        {
-            return _client.TopologyRequest().Send();
-        }
+      partitions: [Array],
+      nodeId: 0,
+      host: 'zeebe-0.zeebe-broker-service.231bb36a-1588-4f1e-b4f6-e09944d7efd7-zeebe.svc.cluster.local',
+      port: 26501
     }
+  ],
+  clusterSize: 1,
+  partitionsCount: 1,
+  replicationFactor: 1
 }
 ```
 
-- Save the file.
-
 </TabItem>
+<TabItem value="java">
+</TabItem>
+</Tabs>
 
-<TabItem value="go">
-
-[Video link](https://youtu.be/_GQuqEwzWKM?t=281)
-
-We will use [GoDotEnv](https://github.com/joho/godotenv) to environmentalize the client connection credentials.
-
-- Add GoDotEnv to the project:
-
-```bash
-go get github.com/joho/godotenv
-```
-
-- Add the client connection credentials for your cluster to the file `.env`:
-
-**Note**: _make sure to remove the `export` keyword from each line_.
+## Create a BPMN model
 
-```
-ZEEBE_ADDRESS='aae86771-0906-4186-8d82-e228097e1ef7.zeebe.camunda.io:443'
-ZEEBE_CLIENT_ID='hj9PHRIiRqT0~qHvFeqXZV-J8fLRfifB'
-ZEEBE_CLIENT_SECRET='.95Vlv6joiuVR~mJDjGPlyYk5Pz6iIwFYmmQyX8yU3xdB1gezntVMoT1SQTdrCsl'
-ZEEBE_AUTHORIZATION_SERVER_URL='https://login.cloud.camunda.io/oauth/token'
-```
+<Tabs groupId="language" defaultValue="nodejs" values={
+[
+{ label: 'NodeJS', value: 'nodejs', },
+{ label: 'Java', value: 'java' },
+] }>
 
-- Save the file.
+<TabItem value="nodejs">
 
 </TabItem>
-
 <TabItem value="java">
 </TabItem>
+</Tabs>
 
-<TabItem value="javaspring">
+- Download and install the [Zeebe Modeler](https://github.com/zeebe-io/zeebe-modeler/releases).
+- Open Zeebe Modeler and create a new BPMN Diagram.
```

Comment:
Sorry I missed this earlier, can you please modify this to link the Camunda Modeler and working with Camunda Modeler instead? Thanks!